bin/cron/push_latest_rds_backup_to_secondary_account

```diff
@@ -98,8 +98,8 @@ def main
 
   # Find the latest automated backup
   sorted_snapshots = production_instance.snapshots.
-      select {|snap| snap.status == 'available' && !snap.snapshot_id.start_with?('temp-snapshot-')}.
-      sort_by(&:snapshot_create_time)
+    select {|snap| snap.status == 'available' && !snap.snapshot_id.start_with?('temp-snapshot-')}.
+    sort_by(&:snapshot_create_time)
```

Comment:
I'm a little confused about this rule. It seems most of the changes here are indenting 4 spaces, but this one is going back from 4 to 2?